cabot/cabotapp/migrations/0017_auto__add_index_statuscheckresult_check_id.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding index on 'StatusCheckResult', fields ['check', u'id']
+        db.create_index(u'cabotapp_statuscheckresult', ['check_id', u'id'])
```

Comment:
@m-brown only question is, should we do `CREATE INDEX CONCURRENTLY` here to avoid downtime?